src/lib/connectTraceToPlot.js

```diff
@@ -31,7 +31,7 @@ export default function connectTraceToPlot(WrappedComponent) {
 
       let fullTrace = {};
       for (let i = 0; i < fullData.length; i++) {
-        if (trace.uid === fullData[i]._fullInput.uid) {
+        if (trace.uid === fullData[i]._fullInput._input.uid) {
```

Comment:
NB this allows us to get at the real input even if `_fullInput` doesn't have the same `uid`

Reply:
should we put a comment in the code for that..?

Reply:
@etpinard any issues you many be aware of with this type of check?